src/pages/components/utilities.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: Utilities
+description: >
+  Utility functions enable advanced theme usage. They let you customize
+  components in a granular way.
+package: '@zendeskgarden/react-theming'
+components:
+  - theming/src/utils/arrowStyles.ts
+  - theming/src/utils/getColor.ts
+  - theming/src/utils/getLineHeight.ts
+  - theming/src/utils/mediaQuery.ts
+  - theming/src/utils/menuStyles.ts
+  - theming/src/utils/retrieveComponentStyles.ts
+  - theming/src/utils/withTheme.ts
+---
+
+## How to use it
+
+### Menu and arrow styles
+
+Use menu and arrow styling utilities to build Garden-styled menu components.
+
+import MenuArrowStyles from '../../examples/utilities/MenuArrowStyles.tsx';
+import MenuArrowStylesCode from '!!raw-loader!../../examples/utilities/MenuArrowStyles.tsx';
+
+<CodeExample code={MenuArrowStylesCode}>
+  <MenuArrowStyles />
+</CodeExample>
+
+## Configuration
+
+import COMPONENTS from '../../examples/utilities/components.ts';
+
+<Configuration reactPackage={props.data.mdx.package} components={COMPONENTS} isTheming />
+
+## API
+
+### arrowStyles
+
+CSS for an arrow at the given position and with the given size. The arrow
+inherits the base element's `border`, `background`, and `box-shadow`. For
+proper `border` and `box-shadow` styling, the element to which arrow styles
+are applied must be wrapped in a positioned box (`relative`, `absolute`, or
+`fixed`) that has a `z-index` greater than or equal to zero.
+
+<PropSheet components={COMPONENTS} componentName="arrowStyles" headerName="Parameter" />
+
+### getColor
+
+Get a color for the given hue, shade, and theme. The color is resolved with a
+fallback algorithm that succeeds whether or not the particular shade exists
+within the theme.
```

Comment:
This wording is confusing to me as a non-developer. I think what it's trying to say is that `getColor` will provide a color even if the particular shade is not part of the theme. 

Consider being more explicit that the shade doesn't have to be in the theme. I think that's what was confusing to me. Also the word "succeeds." Maybe something like "…the algorithm returns a valid color even if the particular shade doesn't exist in the current theme."